app/data_models/questionnaire_store.py

```diff
@@ -75,6 +141,7 @@ def serialize(self) -> str:
         data = {
             "METADATA": self._metadata,
             "ANSWERS": list(self.answer_store),
+            "SUPPLEMENTARY_DATA": self.supplementary_data_store.serialize(),
```

Comment:
I notice now in the questionnaire store within the SUPPLEMENTARY_DATA field we have a supplementary_data key as per the image here - is this nesting necessary?
<img width="1378" alt="Screenshot 2023-06-22 at 14 26 34" src="https://github.com/ONSdigital/eq-questionnaire-runner/assets/47635349/0135ea41-c6cf-4002-a992-b452a248c4cb">

Reply:
Forgot the list_items below, maybe this key could be reamed to "data"?

Reply:
good idea 👍  will go with data instead

Reply:
done